cpp/src/slice2cpp/Gen.cpp

```diff
@@ -117,14 +116,7 @@ writeConstantValue(IceUtilInternal::Output& out, const TypePtr& type, const Synt
         }
         else if(bp && bp->kind() == Builtin::KindLong)
         {
-            if(cpp11)
-            {
-                out << value << "LL";
-            }
-            else
-            {
-                out << "ICE_INT64(" << value << ")";
-            }
+            out << value << "LL";
```

Comment:
Is this correct now that we use int64_t?

Reply:
Hmm... good question. I'll open an issue to investigate.

Reply:
I think it is not correct, LL is valid as long as int64_t is defined as `long long int` or `long long`. 

There is `INT64_C` macro for this, see https://en.cppreference.com/w/cpp/types/integer